### DISCUSSION TITLE
export task -  "StatusMessage": "Task expired and automatically canceled"

### DISCUSSION THREAD

**Discussion by prnr** [aws/aws-cli#8416] [Q&A] [answered]
I used the ec2 create-instance-export-task command to export one instance with a single 8GB volume, and waited for over 100 hours. And today, I checked the progress with the describe-export-task command and found the following results.
No results were detected in any search engine for the error message.

Is there any way to check why it's not progressing?

{
    "ExportTasks": [
        {
            "ExportTaskId": "export-i-00000000000000",
            "ExportToS3Task": {
                "ContainerFormat": "ova",
                "DiskImageFormat": "vmdk",
                "S3Bucket": "backup-00000",
                "S3Key": "vms/export-i-0000000000000.ova"
            },
            "InstanceExportDetails": {},
            "State": "cancelled",
            "StatusMessage": "Task expired and automatically canceled"
        }
    ]
}

**Comment by RyanFitzSimmonsAK** [comment, marked as answer]
I replied to the issue you created (https://github.com/aws/aws-cli/issues/8464), so I'll be closing this discussion.
